### DISCUSSION TITLE
How to see the performance measures for the best performing models for multivariate time series data and how to hide the model training progress

### DISCUSSION THREAD

**Discussion by xsguo1031** [winedarksea/AutoTS#195] [Q&A]
Hi, 

AutoTS is a great package. I have been using it a lot and have two questions:

1. How to hide the model training and validation progress? I'd like to know this answer because I tried to run the model-building process 30 times to test the stability given that there is some randomness in the final forecasts, however, AutoTS automatically displays all the results for all models, which took too much space and crashed my notebook. Therefore, it would be great if there is a parameter we can set to hide the model training and validation progress. 
2. For the multivariate time series data, how to view the performance measures for the best-performing model for each time series sequence, respectively, only? 

Thanks so much!

**Comment by winedarksea** [comment]
@xsguo1031 thanks for the compliment, always a great way to start a request!
1. Try first `verbose=0` to AutoTS() and if that isn't enough try `verbose=-1`. I have the same problem when I use notebooks with all those printed messages, I generally don't use notebooks with AutoTS for that reason.
2. `model.initial_results.per_series_mae` for MAE, and same format for the other metrics that are captured. It will have duplicates for each model, appended one for each validation.  There's also a `model.score_per_series` if you are interested in seeing the metric_weighting generated score.

**Reply by xsguo1031** [reply]
@winedarksea 

Thanks for such a quick and detailed response. 

For the second question, when using model.initial_results.per_series_mae, it shows a dataframe with thousands of rows, even though I only have two backward cross-validation sets. 

So how to understand this dataframe? Which row should be used for the performance measures for the validation results?

Thanks again!

**Reply by winedarksea** [reply]
The results there include every model tested in the cross validation. Find the index that matches model.best_model_id and those are the results for the best model.
